packages/cdktf-cli/bin/cmds/handlers.ts

```diff
@@ -418,6 +423,10 @@ export async function providerAdd(argv: any) {
     console.log(
       "Local providers have been updated. Running cdktf get to update..."
     );
-    await get({ language: language, output: config.codeMakerOutput });
+    await get({
+      language: language,
+      output: config.codeMakerOutput,
+      parallelism: -1,
```

Comment:
Users that have not enough memory would not be able to use the `provider add` command anymore if they can't set this option as get recreates all provider bindings.

Reply:
Then let's default to 1 here 👍